projectDocs/dev/userGuideStandards.md

```diff
@@ -18,13 +18,17 @@ Feature flags, comboboxes and checkboxes should be included using the following
 `FeatureDescriptionAnchor` should not include the settings category.
 Once the anchor is set it cannot be updated, while settings may move categories.
 
-```text2tags
-==== The name of a feature ====[FeatureDescriptionAnchor]
-: Default
-  Enabled
-: Options
-  Default (Enabled), Enabled, Disabled
-:
+```md
+|||
+|-|-|
+| Options | Default (Enabled), Enabled, Disabled |
+| Default | Enabled |
+| Toggle command | `nvda+shift+e` |

```

Comment:
This PR should be focused on format conversion to something compatible with markDown, i.e. converting the existing definition lists to tables. Other changes that can lead to discussion and opinion have not been fully discussed (there are unanswered comments); thus I'd really encourage to handle them in a separate PR or issue.

Thus, I'd recommend to have this addition of the key row discussed in a separate PR or issue. If there is a reason why this should be handled in this same PR, could you please explain in the initial description why?
What happends with toggle keys in the paragraphs describing options where there was no definition list?

Reply:
I've expanded the PR description to document the changes to the standards more.

>What happends with toggle keys in the paragraphs describing options where there was no definition list

When we change standards to the documentation we rarely update all the docs at once. Changes to the standards are just for future work.